opm/autodiff/FlowMain.hpp

```diff
@@ -109,6 +111,101 @@ namespace Opm
     {
         boost::filesystem::path simulationCaseName( const std::string& casename );
         int64_t convertMessageType(const Message::type& mtype);
+
+        namespace fs = boost::filesystem;
```

Comment:
I'd like the ParallelFileMerger class to be moved to a separate file, since it is a separate concept from FlowMain.